docs/antrea-network-policy.md

```diff
@@ -81,37 +81,41 @@ Tiers have the following characteristics:
 Antrea release 0.9.x introduced support for 5 static tiers. These static tiers
 have been removed in favor of Tier CRDs as mentioned in the previous section.
 On startup, antrea-controller will create 5 Read-Only Tier resources
-corresponding to the static tiers for default consumption as shown below.
+corresponding to the static tiers for default consumption, as well as a
+Baseline Tier that will be enforced after developer-created K8s NetworkPolicies.
+The details for these tiers are shown below:
 
 ```
-    Emergency -> Tier name "emergency" with priority "5"
+    Emergency   -> Tier name "emergency" with priority "5"
     SecurityOps -> Tier name "securityops" with priority "50"
-    NetworkOps -> Tier name "networkops" with priority "100"
-    Platform -> Tier name "platform" with priority "150"
+    NetworkOps  -> Tier name "networkops" with priority "100"
+    Platform    -> Tier name "platform" with priority "150"
     Application -> Tier name "application" with priority "250"
+    Baseline    -> Tier name "baseline" with priority "253"
 ```
 
-Any Antrea policy CRD referencing a static tier in its spec will now internally
-reference the corresponding Tier resource, thus maintaining the order of
-enforcement.
+Any Antrea-native policy CRD referencing a static tier in its spec will now internally
+reference the corresponding Tier resource, thus maintaining the order of enforcement.
 
-Previously, the static tiers created were as follows in the relative order of
-precedence:
+The static tier resources are created as follows in the relative order of
+precedence compared to K8s NetworkPolicies:
 
 ```
-    Emergency > SecurityOps > NetworkOps > Platform > Application  
+    Emergency > SecurityOps > NetworkOps > Platform > Application > K8s NetworkPolicy > Baseline
 ```
 
-Thus, all Antrea Policy resources associated with "Emergency" tier will be
-enforced before any other Antrea Policy resource associated with any other
+Thus, all Antrea-native Policy resources associated with "Emergency" tier will be
+enforced before any other Antrea-native Policy resource associated with any other
 tier, until a match occurs, in which case the policy rule's `action` will be
-applied. The "Application" tier carries the lowest precedence, and any Antrea
-Policy resource without a `tier` name set in its spec will be associated with
-the "Application" tier. Even though the policies associated with the
-"Application" tier carry the lowest precedence amongst all the tiers, they are
-still enforced before K8s NetworkPolicies. Thus, admin-created tiered Antrea
-Policy CRDs have a higher precedence than developer-created K8s
-NetworkPolicies.
+applied. **Any Antrea-native Policy resource without a `tier` name set in its spec
+will be associated with the "Application" tier.** Policies associated with the
```

Comment:
Policies associated with the first 5 static, read-only tiers, as well as with all the custom tiers created with a priority value lower than 250 (priority values greater than or equal to 250 are not allowed), will be enforced before K8s NetworkPolicies.